samples/base/Core.cpp

```diff
@@ -104,38 +103,42 @@ namespace darwin {
         // OPTIONS
         log.setLevel(logger::Warning); // Log level by default
         opt = -1;
-        while((opt = getopt(ac, av, ":l:h")) != -1)
+        while((opt = getopt(ac, av, ":l:hno:")) != -1)
         {
-            DARWIN_LOG_DEBUG("OPT : " + std::to_string(opt));
-            DARWIN_LOG_DEBUG("OPTIND : " + std::to_string(optind));
             switch(opt)
             {
+                case 'n':
+                    daemon = false;
+                    break;
+                case 'o':
+                    if (not log.setFilePath(optarg))
+                        return false;
+                    break;
                 case 'l':
                     log_level.assign(optarg);
                     if(!Core::SetLogLevel(log_level)){
-                        DARWIN_LOG_ERROR("Core:: Program Arguments:: Unknown log-level given");
+                        std::clog << "Unknown log-level " << optarg << std::endl << std::endl;
```

Comment:
why use clog instead of cerr?

Reply:
Cannot remember exactly, main difference between clog and cerr being buffering of clog, I guess that was the reason behind my decision
Don't remember why that would be relevant at this moment though. Reflecting back on it, it might be better to revert to cerr

Reply:
change made, I leave you to review/resolve the conversation